### PR TITLE
Pin a Salt version

### PR DESCRIPTION
**Pull request by dpoirier**

### CODE REVIEW

File: fabfile.py [hunk 1]
```diff
@@ -56,24 +58,80 @@ def initialize_env():
...
-        if not installed:
-            sudo('apt-get update -qq -y')
-            sudo('apt-get install python-software-properties -qq -y')
-            sudo('add-apt-repository ppa:saltstack/salt -y')
-            sudo('apt-get update -qq')
-            sudo('apt-get install salt-master -qq -y')
-            sudo('apt-get install python-pip git-core python-git python-gnupg haveged -qq -y')
+        sudo('apt-get update -qq')
+        sudo('apt-get install python-pip git-core python-git python-gnupg haveged -qq -y')
         put(local_path='conf/master.conf',
```
Comment (mlavin) [inline fabfile.py]: Should this go after the install? I feel like the install is going to write this file.

File: fabfile.py [hunk 2]
```diff
@@ -124,7 +171,8 @@ def setup_minion(*roles):
     with open(path, 'w') as f:
         yaml.dump(config, f, default_flow_style=False)
     put(local_path=path, remote_path="/etc/salt/minion", use_sudo=True)
```
Comment (mlavin) [inline fabfile.py]: Same here. I think the config should be put on the server after the install.

File: fabfile.py (hunk 1 above)
Comment (dpoirier) [inline fabfile.py]: I was assuming the install wouldn't overwrite an existing config file, but I should test that.

Comment (dpoirier) [inline fabfile.py]: Confirmed, using the bootstrap to git install the master or minion does not modify an existing config file for either.

Comment (mlavin) [inline fabfile.py]: Seems odd but as long as it works.

**Comment by mlavin** [comment]
:+1:

**Pull request merged**